File: Python/lt_01_array/lt_01_array_01.py
media = 0.0
num = 0
soma = 0

vetor = []

for i in range(0, 50):
    vetor.append(int(input("insira um numero: ")))

for i in vetor:
    if i > 10 and i < 200:
        media += i
        num += 1
    if i % 2 == 1:
        soma += i

if num != 0:
    media /= num

print("\nA soma e", soma, "e a media e", "%2.lf" % media, ".\n")

File: C/lt_01_array/lt_01_array_01.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
  int vet[50], i, num, soma = 0;
  double media = 0;
  srand(time(NULL));

  for (i = 0; i < (sizeof(vet) / sizeof(vet[0])); i++) {
    /*printf("Insira um numero: ");
    scanf("%d", &vet[i]);*/
    vet[i] = rand() % 100;
  }

  for (i = 0; i < (sizeof(vet) / sizeof(vet[0])); i++) {
    printf("|%d", vet[i]);
    if ((vet[i] > 10) && (vet[i] < 200)) {
      media += vet[i];
      num++;
    }
    if (vet[i] % 2 == 1) {
      soma += vet[i];
    }
  }
  printf("|\n");
  media /= num;

  printf("A media dos valores e %.2lf.\nA soma dos inpares e %d.\n", media,
         soma);
  return 0;
}
